code/modules/atmospherics/machinery/components/fusion/hfr_parts.dm

```diff
@@ -11,6 +11,7 @@
 	anchored = TRUE
 	density = TRUE
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+	flags_1 = PREVENT_CONTENTS_EXPLOSION_1
```

Comment:
This should probably be standardized so that any INDESTRUCTIBLE machinery with component parts requires this flag.

Reply:
Could you like clarify by what "standardized" means there, so as in automatically add the flag to all INDESTRUCTIBLE machinery during initialisation?

Reply:
Either that or unit test that checks and warns of any path that doesn't have it.